mailbox/cassandra/src/main/java/org/apache/james/mailbox/cassandra/mail/CassandraAttachmentDAOV2.java

```diff
@@ -141,7 +141,7 @@ public CassandraAttachmentDAOV2(BlobId.Factory blobIdFactory, Session session) {
     private PreparedStatement prepareDelete(Session session) {
         return session.prepare(
             QueryBuilder.delete().from(TABLE_NAME)
-                .where(eq(ID, bindMarker(ID))));
+                .where(eq(ID_AS_UUID, bindMarker(ID_AS_UUID))));
```

Comment:
I don't understand this change: you changed the query but nothing breaks?

Reply:
That's a fixup of a mistake I did